examples/Python/PET/scatter_simulation.py

```diff
@@ -0,0 +1,173 @@
+
+There is an interactive demo with much more documentation on this process.
+You probably want to check that instead.
+'''
+
+## CCP PETMR Synergistic Image Reconstruction Framework (SIRF)
+## Copyright 2019 University of Hull
+##
+## This is software developed for the Collaborative Computational
+## Project in Positron Emission Tomography and Magnetic Resonance imaging
+## (http://www.ccppetmr.ac.uk/).
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+##   you may not use this file except in compliance with the License.
+##   You may obtain a copy of the License at
+##       http://www.apache.org/licenses/LICENSE-2.0
+##   Unless required by applicable law or agreed to in writing, software
+##   distributed under the License is distributed on an "AS IS" BASIS,
+##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+##   See the License for the specific language governing permissions and
+##   limitations under the License.
+
+__version__ = '0.1.0'
+from docopt import docopt
+args = docopt(__doc__, version=__version__)
+
+# import engine module
+exec('from sirf.' + args['--engine'] + ' import *')
+
+from pUtilities import show_2D_array
+
+# import engine module
+exec('from sirf.' + args['--engine'] + ' import *')
+
+# process command-line options
+data_file = args['--file']
+data_path = args['--path']
+if data_path is None:
+    data_path = examples_data_path('PET')
+raw_data_file = existing_filepath(data_path, data_file)
+addv = float(args['--addv'])
+back = float(args['--back'])
+beff = 1/float(args['--norm'])
+output_file = args['--output']
+
+def main():
+
+##    AcquisitionData.set_storage_scheme('mem')
+
+    # no info printing from the engine, warnings and errors sent to stdout
+    msg_red = MessageRedirector()
+    # output goes to files
+    ##    msg_red = MessageRedirector('info.txt', 'warn.txt', 'errr.txt')
+
+    # # create an empty image
+    # image = ImageData()
+    # image_size = (31, 111, 111)
+    # voxel_size = (3.375, 3, 3) # voxel sizes are in mm
+    # image.initialise(image_size, voxel_size)
+    #
+    # # create a shape
+    # shape = EllipticCylinder()
+    # shape.set_length(400)
+    # shape.set_radii((40, 100))
+    # shape.set_origin((10, 60, 0))
+    #
+    # # add the shape to the image
+    # image.add_shape(shape, scale = 1)
+    #
+    # # add another shape
+    # shape.set_radii((30, 30))
+    # shape.set_origin((10, -30, 60))
+    # image.add_shape(shape, scale = 1.5)
+    #
+    # # add another shape
+    # shape.set_origin((10, -30, -60))
+    # image.add_shape(shape, scale = 0.75)
+    #
+    # # z-pixel coordinate of the xy-crossection to show
+    # z = int(image_size[0]/2)
+    #
+    # # show the phantom image
+    # image_array = image.as_array()
+    # show_2D_array('Phantom image', image_array[z,:,:])
+
+    # Create a template Acquisition Model
+    tmpl_acq_data = pystir.cSTIR_acquisitionsDataFromScannerInfo('ECAT 931', 1, 0, 1)
```

Comment:
This line will obviously have to be replaced before the PR gets merged.

Reply:
I see that in other examples you read the ProjDataInfo from a file. Is there another way? 
How would you change it?

Reply:
Which other examples does it exist in? Our python wrapping code should be the only code calling anything with `pystir`. 

https://github.com/CCPPETMR/SIRF/blob/ea4a56f9f07ae5a17d74887fbca002dc69313b5c/src/xSTIR/pSTIR/STIR.py#L551-L583

The attached code snipped shows that the constructor of `AcquisitionData` accepts a scanner name as input and calls `cSTIR_acquisitionsDataFromScannerInfo` itself. So I think you should simply be able to use: 

```
tmpl_acq_data = AcquisitionData('ECAT 931', 1, 0, 1)
```

Reply:
:ok:

Reply:
You absolutely must have it the way Richard suggested - `pystir` objects are not to be used directly.